build.gradle

```diff
@@ -302,8 +302,7 @@ dependencies {
     liquibaseRuntime 'org.postgresql:postgresql:42.2.7'
     liquibaseRuntime 'ch.qos.logback:logback-classic:1.2.3'
 
-    testImplementation group: 'junit', name: 'junit', version: '4.13.2'
-    testImplementation group: 'org.hamcrest', name: 'hamcrest', version: '2.1'
+    testImplementation group: 'org.junit.vintage', name: 'junit-vintage-engine', version: '5.3.0-M1'
```

Comment:
This dependency enables JUnit 5 to run JUnit 4 (and 3) tests without change. Once all tests have been migrated to 5, it can be removed.